Modix.Bot/Modules/UserInfoModule.cs

```diff
@@ -21,20 +23,22 @@ public class UserInfoModule : ModuleBase
         private const string Format = "{0}: {1} ago ({2:yyyy-MM-ddTHH:mm:ssK})\n";
 
         //optimization: UtcNow is slow and the module is created per-request
-        private readonly DateTime _utcNow = DateTime.UtcNow; 
+        private readonly DateTime _utcNow = DateTime.UtcNow;
 
-        public UserInfoModule(ILogger<UserInfoModule> logger, IUserService userService, IModerationService moderationService, IAuthorizationService authorizationService)
+        public UserInfoModule(ILogger<UserInfoModule> logger, IUserService userService, IModerationService moderationService, IAuthorizationService authorizationService, IMessageRepository messageRepository)
         {
             Log = logger ?? new NullLogger<UserInfoModule>();
             UserService = userService;
             ModerationService = moderationService;
             AuthorizationService = authorizationService;
+            MessageRepository = messageRepository;
         }
 
         private ILogger<UserInfoModule> Log { get; }
         private IUserService UserService { get; }
         private IModerationService ModerationService { get; }
         private IAuthorizationService AuthorizationService { get; }
+        private IMessageRepository MessageRepository { get; }
```

Comment:
Why private properties?

Reply:
It's a simple and easy way to avoid prefixing private members with `_` or `this.`.